repair_quotation_manual_sync/models/sale_order.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2025 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    repair_sync_warning = fields.Boolean(compute="_compute_repair_sync_warning")
+    repair_sync_message = fields.Char(compute="_compute_repair_sync_warning")
+
+    @api.depends(
+        "repair_order_ids.move_ids.invoiceable",
+        "repair_order_ids.move_ids.sale_line_id",
+        "repair_order_ids.move_ids.product_uom_qty",
+        "repair_order_ids.move_ids.sale_line_id.product_uom_qty",
+    )
+    def _compute_repair_sync_warning(self):
+        for order in self:
+            has_sync_issue = False
+            for repair in order.repair_order_ids:
+                for move in repair.move_ids:
+                    if move.invoiceable:
+                        if not move.sale_line_id and move.product_uom_qty > 0:
+                            has_sync_issue = True
+                            break
+                        if (
+                            move.sale_line_id
+                            and move.product_uom_qty
+                            != move.sale_line_id.product_uom_qty
+                        ):
+                            has_sync_issue = True
+                            break
+                if has_sync_issue:
+                    break
+            if has_sync_issue:
+                order.repair_sync_warning = True
+                order.repair_sync_message = (
+                    "Repair Order and Quotation are not synchronized. "
+                    "Would you like to synchronize this Sale Order "
+                    "with the repair lines?"
+                )
```

Comment:
```suggestion
                order.repair_sync_message = (
                    "Repair Order and Quotation are not synchronized. "
                    "Would you like to synchronize this Sale Order "
                    "with the repair lines?"
                )
```
As the text is static, and it doesn't depend on any value from the `sale.order`, we could remove the field and declare the static text directly in the view.